src/views/CharacterListView.js

```diff
@@ -25,11 +28,21 @@ class CharacterListView extends React.Component {
   }
 }
 
+
+
 // our mapStateToProps needs to have two properties inherited from state
+const mapStateToProps = state => {
+  return {
+    characters: state.characters,
+    fetching: state.fetching
```

Comment:
check your DM - it finally dawned on me what was going on with the reducers here.

Reply:
got a chance to read it! thank you! i'll be making the proper modifications this afternoon.